bzt/bza.py

```diff
@@ -728,20 +728,11 @@ def stop(self):
     def start(self):
         self._request(self.address + '/api/latest/proxy/recording/start', method='POST')
 
-    def get_jmx(self):
-        response = self._request(self.address + '/api/latest/proxy/download?format=jmx&smart=false', raw_result=True)
-        return response
-
-    def get_smart_jmx(self):
-        # wait for availability
-        while True:
-            response = self._request(self.address + '/api/latest/proxy')
-            if response['result']['smartjmx'] == "available":
-                break
-            time.sleep(self.delay)
-
-        response = self._request(self.address + '/api/latest/proxy/download?format=jmx&smart=true', raw_result=True)
-        return response
+    def get_jmx(self, smart=False):
+        url = '/api/latest/proxy/download?format=jmx&smart=' + str(smart).lower()
```

Comment:
How will this work with smartjmx which might take long time to generate?

Reply:
Now it's generating it when you press the button and it's blocking